assets/src/room.ts

```diff
@@ -154,25 +153,25 @@ export class Room {
   }
 
   public init = async () => {
-    try {
-      this.localAudioStream = await navigator.mediaDevices.getUserMedia(
-        AUDIO_MEDIA_CONSTRAINTS
-      );
-    } catch (error) {
-      console.error("Error while getting local audio stream", error);
-    }
+    const hasVideoInput: boolean = (
+      await navigator.mediaDevices.enumerateDevices()
+    ).some((device) => device.kind === "videoinput");
+
+    // Ask browser for permissions
```

Comment:
I would say that this is more `// Ask user for permissions`  :stuck_out_tongue_closed_eyes: